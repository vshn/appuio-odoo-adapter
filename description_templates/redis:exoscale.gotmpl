Exoscale Redis

{{template "_dbaas_usage" . -}}
