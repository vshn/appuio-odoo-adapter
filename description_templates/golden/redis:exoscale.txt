Exoscale Redis

Qty: 87955674 Instance-Hours
Unit Price: CHF 0.00000075 / UNIT / Hour
